### DISCUSSION TITLE
Bforartists 3 version 3.2.2

### DISCUSSION THREAD

**Discussion by ghost** [Bforartists/Bforartists#3185] [General]
Hello friends,

Bforartists 3 Version 3.2.2 is a maintenance release to keep Bforartists updated with the latest changes from Blender 3.2.2

Splashscreen is kindly provided by **Seanr**

The binary downloads are attached. Yet per usual they can also be found in the download section of the official website.
https://www.bforartists.de/download/

The release note can be found here:
https://www.bforartists.de/release-notes/

Thank you!

Draise, Reiner, Iyad and the rest of the Team

![Screenshot from 2022-08-07 11-58-36](https://user-images.githubusercontent.com/25552173/183285565-aacb5fe2-5b76-4b69-a2ca-81bf7622b238.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/Bforartists/Bforartists/releases/tag/v3.2.2'>Bforartists 3 version 3.2.2</a>.</em>

**Comment by ReinerBforartists** [comment]
You forgot to mention yourself as a contributor :)
